### PR TITLE
Dp 18775 info details featured flag

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Added new checkbox and updated permissions accordingly.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-18775

### CODE REVIEW

File: docroot/themes/custom/mass_admin_theme/js/conditional-fields.js [hunk 1]
```diff
@@ -141,4 +141,29 @@
...
+   * Enable/disable fields on info details.
+   *
+   * Hide or show the banner field (field_banner_image)
+   * based on the field_details_enable_fields value.
+   */
+  Drupal.behaviors.enableInfoFieldsConditional = {
+    attach: function (context) {
+      $('.field--name-field-details-enable-fields', context).change(function () {
+        if ($(this).find('input').prop('checked') === false) {
+          // $(this).siblings('.field--name-field-banner-image').hide().find('.fieldgroup').removeAttr('required');
```
Comment (tom-fleming) [inline docroot/themes/custom/mass_admin_theme/js/conditional-fields.js]: @arthurbaghdas Do we need this commented out code?

File: docroot/themes/custom/mass_admin_theme/js/conditional-fields.js [hunk 2]
```diff
@@ -141,4 +141,29 @@
...
+          $(this).siblings('.field--name-field-banner-image').hide();
+          $(".field--name-field-info-details-sections details#section-content .field--name-field-section-long-form-content input[id$='-subform-field-section-long-form-content-add-more-add-more-button-callout-link']").hide();
+          $(".field--name-field-info-details-sections details#section-content .field--name-field-section-long-form-content input[id$='-subform-field-section-long-form-content-add-more-add-more-button-info-details-card-group']").hide();
+        }
+        else {
+          // $(this).siblings('.field--name-field-tabl-alignment').show().find('.fieldgroup').attr('required', 'required');
```
Comment (tom-fleming) [inline docroot/themes/custom/mass_admin_theme/js/conditional-fields.js]: @arthurbaghdas Do we need this commented out code?

File: changelogs/DP-18775.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Added:
+  - description: new checkbox added to Info details Content type to toggle other Banner, Callout and Card group visibility..
```
Comment (tom-fleming) [inline changelogs/DP-18775.yml]: @arthurbaghdas Not sure where this ends up getting published, but could we clean up this text a bit (mostly the second period)? 

"Added checkbox to Info details Content type to toggle other Banner, Callout and Card group visibility."

Comment (arthurbaghdas) [reply changelogs/DP-18775.yml]: Yes sure, thank you

Comment (arthurbaghdas) [reply changelogs/DP-18775.yml]: This is going to be published in the release process to CHANGELOG.md file

File: docroot/themes/custom/mass_admin_theme/js/conditional-fields.js (hunk 2 above)
Comment (arthurbaghdas) [reply docroot/themes/custom/mass_admin_theme/js/conditional-fields.js]: No we do not need, I've just copied from top for reference and forgot to clean up my code. Thank you

File: docroot/themes/custom/mass_admin_theme/js/conditional-fields.js (hunk 1 above)
Comment (arthurbaghdas) [reply docroot/themes/custom/mass_admin_theme/js/conditional-fields.js]: No we do not need, I've just copied from top for reference and forgot to clean up my code. Thank you

**Comment by arthurbaghdas** [comment]
@tom-fleming please review again, I've cleaned up the code

**Review by tom-fleming — APPROVED** [review]

**Pull request merged**